spec/lib/open_food_network/i18n_config_spec.rb

```diff
@@ -5,6 +5,14 @@
 
 module OpenFoodNetwork
   describe I18nConfig do
+    before do
+      # These are only needed here because we are stubbing ENV elsewhere in this file
+      #   Here we make these calls to ENV return a value and that makes database_cleaner work
+      allow(ENV).to receive(:[]).with("DATABASE_CLEANER_ALLOW_PRODUCTION").and_return("false")
+      allow(ENV).to receive(:[]).with("DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL").
+        and_return("false")
```

Comment:
We can replace these line with a default like this:
```ruby
allow(ENV).to receive(:[]).and_call_original
```
and all the subsequent places where we stub a specific response will still work (and the error will be fixed).